src/tools/sdk/dart_sdk_archive/lib/src/version_selector.dart

```diff
@@ -193,15 +194,27 @@ class VersionSelector {
               }
             }
 
+            String baseFileName = '${archiveMap[pa]}-${archiveMap[name]}-'
+                '${archiveMap[platformVariant.architecture]}';
+
+            if (pa == 'Debian package') {
+              // Debian packages start with 2.0.0
+              if (versionInfo.version < Version(2, 0, 0)) {
```

Comment:
Should probably store that version in a top-level final. So it's not created for every entry

Reply:
You mean `Version(2, 0, 0)`? That's only used in this single place.